src/cloud-api-adaptor/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG IMAGE=kindest/node
```

Comment:
optional enhancement would be move it to podvm/Dockerfile.podvm.docker

Reply:
@bpradipt - what was your thoughts on this comment as it seems to make sense to me if it's not too much work?

Reply:
Sorry I forgot to respond. I kept the layout similar for the aws and azure podvm image (ie under <provider>/image).
Right now I would prefer it that way.